decidim-debates/db/migrate/20200716143929_add_endorsable_to_debates.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddEndorsableToDebates < ActiveRecord::Migration[5.2]
+  def change
+    add_column :decidim_debates_debates, :debate_endorsements_count, :integer, null: false, default: 0
+    add_column :decidim_debates_debates, :endorsements_count, :integer, null: false, default: 0
```

Comment:
Why two columns for endorsements count? `Endorsable` only defines the `endorsements_count` counter cache..